lib/fetchers/helper.ts

```diff
@@ -50,3 +50,9 @@ export const getServiceUrlWithParams = ({
   const urlParams = params ? `?${buildParams(params as FetchParams)}` : ""
   return `${baseUrl}${url}${urlParams}`
 }
+
+export const filterFalsyValuesFromArray = <T>(
+  array: (T | undefined | null | false | 0 | "")[]
+): T[] => {
+  return array.filter(Boolean) as T[]
+}
```

Comment:
I'd say that this is so global that you could consider moving it to a global place. Like `lib/helpers/arrays.ts`?
What do you think?

Reply:
I did that in a later commit 😉